edx_toggles/toggles/internal/base.py

```diff
@@ -37,3 +37,12 @@ def get_instances(cls):
         Return the list of class instances sorted by name.
         """
         return sorted(cls._class_instances, key=lambda instance: instance.name)
+
+    def is_disabled(self):
+        return not self.is_enabled()
+
+    def is_toggle_on(self):
+        return self.is_enabled()
+
+    def is_toggle_off(self):
+        return self.is_disabed()
```

Comment:
Ok

Reply:
@robrap why is the test still failing pls ?

Reply:
Looks like this pylint failure:

> edx_toggles/tests/test_waffle.py:43:0: C0303: Trailing whitespace (trailing-whitespace)

Reply:
@Theo-flux: You just need to remove the whitespace from the end of the line. See the failure message above. Can you take care of this?

Also, some of your commit messages might be failing because you are using "feat:xxx" instead of "feat: xxx". You probably need the space after the colon.

Reply:
Okay, I will attend to this.

Reply:
@robrap  Thanks for your patience . To everyone I am very sorry for the delay. I have pushed the necessary updates. Thanks.